src/routes/Agreements.js

```diff
@@ -50,12 +50,11 @@ class Agreements extends React.Component {
     orgs: {
       type: 'okapi',
       path: 'erm/org',
-      records: 'results',
       limitParam: 'perPage',
       perRequest: 100,
-      recordsRequired: '1000',
       params: {
-        stats: 'true',
+        match: 'name',
+        term: '${orgNameFilter}', // eslint-disable-line no-template-curly-in-string
```

Comment:
Same here: `%{orgNameFilter}` makes the same substitution as `${orgNameFilter}` and avoids the wrath of the eslint gods.